app/mailers/owners_mailer.rb

```diff
@@ -9,15 +9,20 @@ class OwnersMailer < ApplicationMailer
   default_url_options[:host] = Gemcutter::HOST
   default_url_options[:protocol] = Gemcutter::PROTOCOL
 
-  def ownership_confirmation(ownership)
```

Comment:
Would you also revert to using the ownership record here instead of id? It looks like when you enqueue, the ownership record definitely exists so you could save the not found guard.

Reply:
ok, reverted